src/app/header/header.component.ts

```diff
@@ -30,4 +30,8 @@ export class HeaderComponent implements OnInit {
   myDashBoard(){
     this.router.navigate(["/"])
   }
+
+  createTeam(){
+    this.router.navigate(['/CreateTeam', this.authService.getTeamId()]);
```

Comment:
To create a new team we don't need to send team id. It's only needed for updating the teams details. 

The code is sent without testing the full functionality.